lib/lz4.c

```diff
@@ -1326,7 +1326,13 @@ int LZ4_compress_destSize(const char* src, char* dst, int* srcSizePtr, int targe
 
 LZ4_stream_t* LZ4_createStream(void)
 {
+#if (LZ4_HEAPMODE)
     LZ4_stream_t* const lz4s = (LZ4_stream_t*)ALLOC(sizeof(LZ4_stream_t));
+#else
+    static LZ4_stream_t lz4sBody;
```

Comment:
`static` is unlikely to be correct here.
This will make the function impossible to multi-thread.